README.md

```diff
@@ -55,13 +55,13 @@ Groups are the building block of Groupie.  An individual `Item` (the unit which
 
 Kotlin
 ```kotlin
-groupAdapter += HeaderItem()
-groupAdapter += CommentItem()
+groupAdapter.add(HeaderItem())
+groupAdapter.add(CommentItem())
 
 val section = Section()
 section.setHeader(HeaderItem())
 section.addAll(bodyItems)
-groupAdapter += section
+groupAdapter.add(section)
```

Comment:
AFAICT this part of the readme was wrong, given it didn't talk about depending on `groupie-kotlin-android-extensions` previously, where these extensions were originally available.